src/mumble/AudioInput.h

```diff
@@ -6,6 +6,7 @@
 #ifndef MUMBLE_MUMBLE_AUDIOINPUT_H_
 #define MUMBLE_MUMBLE_AUDIOINPUT_H_
 
+#include <QtCore/QElapsedTimer>
```

Comment:
```suggestion
#include QElapsedTimer>
```
This is actually the preferred way of including Qt headers. We only found out about that very recently and therefore the rest of the code base partially does use the other style.

Reply:
If clang-format is happy with it, we are too :)
Did you run clang-format yet? If not, pull the latest changes and then just run it on all files that you have edited. That should take care of all formatting aspects.

Reply:
Ran clang-format now. Nice stuff.